irohad/torii/processor/transaction_processor_impl.hpp

```diff
@@ -67,6 +68,33 @@ namespace iroha {
 
       logger::LoggerPtr log_;
 
+      using VerifiedProposalSubscription =
+          subscription::SubscriberImpl<EventTypes,
+                                       SubscriptionDispatcher,
+                                       bool,
+                                       simulator::VerifiedProposalCreatorEvent>;
+      using BlockSubscription = subscription::SubscriberImpl<
```

Comment:
This code is duplicated at least in 4 places. Please unify.

Reply:
https://imgur.com/3l4KntA.png